manifests/board-image/bianbu-desktop-spacemit-k1-sd/2.1.1.toml

```
@@ -0,0 +1,32 @@
+format = "v1"
+
+[metadata]
+desc = "Official bianbu desktop image version 2.1.1"
```

Please make the sentence more natural.

```suggestion
desc = "Official Bianbu Desktop 2.1.1 for SpacemiT K1, SD image"
```

In the future, the word "official" should ideally be phased out, because in the RISC-V world there's hardly any one entity that's "official" for anything. For later additions "Foo Bar 1.2.3 for Baz, provided by SomeCorporation" could be more appropriate, but it's fine for this PR.